rustc-dev-guide/index.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Redirecting to https://rust-lang.github.io/rustc-dev-guide/</title>
+<meta http-equiv="refresh" content="0; URL=https://rust-lang.github.io/rustc-dev-guide/">
```

Comment:
Link updated to use https://rustc-dev-guide.rust-lang.org/ instead

```suggestion
<meta http-equiv="refresh" content="0; URL=https://rustc-dev-guide.rust-lang.org/">
```